subxt/src/storage/utils.rs

```diff
@@ -6,17 +6,12 @@
 //! aren't things that should ever be overridden, and so don't exist on
 //! the trait itself.
 
-use subxt_metadata::StorageHasher;
-
 use super::StorageAddress;
-use crate::{
-    error::{Error, StorageAddressError},
-    metadata::Metadata,
-};
+use crate::{error::Error, metadata::Metadata};
 
 /// Return the root of a given [`StorageAddress`]: hash the pallet name and entry name
 /// and append those bytes to the output.
-pub(crate) fn write_storage_address_root_bytes<Address: StorageAddress>(
+pub fn write_storage_address_root_bytes<Address: StorageAddress>(
```

Comment:
utils wasn't exporting anything before, so now the module is not exported at all so these fns don't need to be pub(crate) to hide themselves any more.

I moved the added fns that were only used in one place into the place they were used.